examples/kubeflow/mnist_with_tfjob/README.md

```diff
@@ -0,0 +1,194 @@
+Now that our data is checked in, it's time to deploy some code. In the same directory run the following:
+
+`git clone https://github.com/pachyderm/pachyderm.git && cd pachyderm/examples/kubeflow/mnist-tfjob`
+
+Next, let's take a look at how things are working under the hood. Below is a snippet of `tfjob.py`
+
+```
+# this is the Pachyderm repo & branch we'll copy files from
+input_bucket = os.getenv('INPUT_BUCKET', 'master.inputrepo')
+# this is the Pachyderm repo & branch  we'll copy the files to
+output_bucket  = os.getenv('OUTPUT_BUCKET', "master.outputrepo")
+# this is local directory we'll copy the files to
+data_dir  = os.getenv('DATA_DIR', "/data")
+
+def main(_):
+  # Declaring Pachyderm S3 Gateway input and output repos.
+  input_url = 's3://' + args.inputbucket + "/data/"
+  output_url = 's3://' + args.outputbucket + "/data/"
+  
+  os.makedirs(args.datadir)
+
+  # first, we copy files from pachyderm into a convenient
+  # local directory for processing.  The files have been
+  # placed into the inputpath directory in the s3path bucket.
+  print("walking {} for copying files".format(input_url))
+  for dirpath, dirs, files in file_io.walk(input_url, True):
+    for file in files:
+      uri = os.path.join(dirpath, file)
+      newpath = os.path.join(args.datadir, file)
+      print("copying {} to {}".format(uri, newpath))
+      file_io.copy(uri, newpath, True)
+
+  path = '/tmp/data/mnist.npz'
+  (train_images, train_labels), (test_images, test_labels) = tf.keras.datasets.mnist.load_data(path)
+
+  <<<< .... MNIST example below ..... >>>>
+```
+The comments in the code provide a pretty good description of what's going on line by line. However a quick breakdown is this:  
+
+We're copying mnist.npz from our Pachyderm repo inputrepo@master via the [S3 Gateway](http://docs.pachyderm.com/en/latest/enterprise/s3gateway.html#using-the-s3-gateway) into a local directory in the container (`/tmp/data/`). Then we tell [TensorFlow](https://www.tensorflow.org/) to load that data and start training. 
+
+#### Mnist. Mnist. Mnist.  
+
+Once our code trains the model, it needs to save it somewhere. Just like we copied data into the container we can copy it back out again, all the while still maintaining some provenance. If you take a look at the `tfjob_mist.py` and scroll towards the bottom you'll see that we're just crawling the same `/tmp/data/` directory and copying it to the Pachyderm S3 Gateway output `url:s3://<pachyderminstance>/master.outputrepo:/data/`
+
+
+```
+print("walking {} for copying to {}".format(args.datadir, output_url))
+  for dirpath, dirs, files in os.walk(args.datadir, topdown=True):   
+    for file in files:
+      uri = os.path.join(dirpath, file)
+      newpath = output_url + file
+      print("copying {} to {}".format(uri, newpath))
+      file_io.copy(uri, newpath, True)
+```
+
+Next, let's move onto how we deploy our code. Start by taking a look at the `tf_job_s3_gateway.yaml`:
```

Comment:
...the `tf_job_s3_gateway.yaml` file: (add _file_)
or
...`tf_job_s3_gateway.yaml`: (remove _the_)